tests/drivers/adc/adc_accuracy_test/testcase.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  tags:
+    - adc
+    - drivers
+  depends_on:
+    - adc
+tests:
+  drivers.adc.accuracy.dac_source:
+    depends_on:
+      - dac
+    harness_config:
+      fixture: dac_adc_loopback
+    extra_configs:
+      - CONFIG_DAC=y
```

Comment:
Why not set this Kconfig in a board overlay? I ask because setting this in `frdm_k64f.conf` would (IMO) make it easier for users to run this test locally, since they don't have to check the `testcase.yaml` to see what Kconfigs are needed